docs/production/custom-domains.md

```diff
@@ -1,28 +1,18 @@
 # Custom Domains (with certs via Let's Encrypt)
 
-Our former constraint of <= 15 certs is no longer applicable! Yay.
-
-https://www.notion.so/darklang/Custom-domains-take-2-c9f04210ec94422695f91bc870bf117e
-
 ## Customer requirements
 
 - You need to set up a CNAME from your desired domain to
-  `<canvas>.builtwithdark.com`.
+  `<canvas>.darkcutsomdomain.com`.
```

Comment:
is `darkcutsomdomain` short for something? dark cuts om domain?

Reply:
lol

Reply:
:hocho: cut some domain